app/src/androidTest/java/org/dhis2/usescases/searchte/SearchTETest.kt

```diff
@@ -124,6 +133,34 @@ class SearchTETest : BaseTest() {
         }
     }
 
+    @Test
+    fun shouldSuccessfullyShowMapAndTeiCard() {
+        val firstName = "Gertrude"
+
+        prepareChildProgrammeIntentAndLaunchActivity()
+
+        searchTeiRobot {
+            clickOnOptionMenu()
+            clickOnShowMap()
+            swipeCarouselToLeft()
+            checkCarouselTEICardInfo(firstName)
+            try {
+                mapIdlingResource = MapIdlingResource(rule)
+                IdlingRegistry.getInstance().register(mapIdlingResource)
+                map = mapIdlingResource!!.map
+            } catch (ex: IdlingResourceTimeoutException) {
+                throw RuntimeException("Could not start test")
+            }
+        }
+    }
```

Comment:
Do you want to assert something on map ?
is it init? and styles? just thinking out loud

I think you can remove the try/catch? Each test has its own instrumentation process, therefore if it crashes the test fails, but the next test continue

Reply:
This PR is not a proper test of the map. I was just setting up the IdlingResource to get the map's data. I concur that we can remove the try/catch.